.coveragerc

```diff
@@ -27,6 +27,9 @@ exclude_lines =
     # Don't complain if non-runnable code isn't run:
     ^if __name__ == ['"]__main__['"]:$
 
+    # Don't complain about type hints
+    ^if TYPE_CHECKING:$
```

Comment:
you'll need to use `if False` since python3.5 is still within our support (for now) -- fortunately `coverage` knows about `if False` by default so you won't need to edit this file after all!